compiler/modules/nback/build.bal

```diff
@@ -349,18 +349,18 @@ function buildReprValue(llvm:Builder builder, Scaffold scaffold, bir:Operand ope
             return [repr, check buildConstString(builder, scaffold, value)];
         }
         else if value == () {
-            return [REPR_NIL, buildConstNil()];
+            return [REPR_NIL, constNil(scaffold)];
         }
         else if value is boolean {
-            return [REPR_BOOLEAN, llvm:constInt(LLVM_BOOLEAN, value ? 1 : 0)];
+            return [REPR_BOOLEAN, constBoolean(scaffold, value)];
         }
         else if value is int {
             boolean alwaysInImmediateRange = IMMEDIATE_INT_MIN <= value && value <= IMMEDIATE_INT_MAX;
             IntRepr repr = { constraints: { min: value, max: value, all: true }, alwaysInImmediateRange };
-            return [repr, llvm:constInt(LLVM_INT, value)];
+            return [repr, constInt(scaffold, value)];
         }
         else if value is float {
-            return [REPR_FLOAT, llvm:constFloat(LLVM_DOUBLE, value)];
+            return [REPR_FLOAT, constDouble(scaffold, value)];
```

Comment:
My idea for the naming scheme was `constFoo` would build something of type `LLVM_FOO` (ex: `constInt` -> `LLVM_INT`). In retrospect, I feel using `LLVM_FLOAT` may be a better choice than `LLVM_DOUBLE`, so it will be similar to `LLVM_INT`, `LLVM_BOOLEAN` etc. (the name is derived from what ballerina type they represent over what llvm type they are using). May I change `LLVM_DOUBLE` to `LLVM_FLOAT` as well or just change this to `constFloat`?

Reply:
The llvm module is naming things from an LLVM perspective. The nback module is naming things from the Ballerina perspective.  So the llvm module should use DOUBLE (because that is what LLVM calls it), but the nback module should call it FLOAT (because that's what Ballerina calls).  So just change this module.

Reply:
Fixed by changing `LLVM_DOUBLE` to `LLVM_FLOAT` (`LLVM_DOUBLE` was a type alias defined in [`scaffold.bal`](https://github.com/ballerina-platform/nballerina/blob/016c743511e152dc889e9e6b11c3481c742fbed3/compiler/modules/nback/scaffold.bal#L12) for `double`) and `constDouble` to `constFloat`